main.tex

```diff
@@ -594,15 +592,15 @@ \subsection{The special case of \(N=2\)}\label{sec:two_individuals}
 conclusions about zero determinant strategies however it is dependent on a
 specific structure of strategies. The analysis carried out here makes no
 assumptions about the structure of the strategies by using actual simulation
-interactions.
+interactions. \cite{Stewart...}
 
-In the next sections close attention to
-strategies who are strong invaders/resistors is given.
+% In the next sections close attention to
```

Comment:
Well it's just "in the next section X" and then the next section opens with "In this section X" in some cases, which was redundant